cmake/Findnl-3.cmake

```diff
@@ -0,0 +1,31 @@
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+# Copyright (c) 2019 Intel Corporation
```

Comment:
2019?

Just to be sure: this is really (c) Intel, you didn't copy it from somewhere on the interwebs?

Reply:
Not copied from anywhere. It is our style of Find modules :)